elementary/utils/schema.py

```diff
@@ -18,12 +18,15 @@ def _load_var_to_dict(var: Union[str, dict]) -> dict:
 
     @staticmethod
     def _load_var_to_list(var: Union[str, list]) -> list:
-        if not var:
+        if not var & type(var) != dict:
```

Comment:
```suggestion
        if not var:
```
I don't think this one should be changed.
Currently we support Union[str, list] so if someone is passing a dict it should be OK to fail 🙂.